changes/probe.py

```diff
@@ -58,11 +68,20 @@ def has_changelog():
 
 
 def has_readme():
-    """README.md"""
+    """README"""
+    readme = None
+    message = 'Create a README file'
+    for filename in os.listdir("."):
+        readme = re.match(r"README($|\.\w{2,})", filename, re.I)
```

Comment:
Instead of iterating through all the files in the toplevel, and then checking against the extensions, what about generating a list of valid READMEs and then checking if they exist?
Something like:
```python
any([exists('README{0}'.format(ext)) for ext in README_EXTENSIONS])
```
?